src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -59,13 +46,9 @@
     <Compile Include="$(MessagePackRoot)SequenceReaderExtensions.cs" LinkBase="BlazorPack\MessagePack" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(BuildNodeJS)' != 'false'">
+  <ItemGroup>
     <!-- We need .Browser.JS to build first so we can embed its .js output -->
-    <EmbeddedResource Include="..\..\Browser.JS\src\dist\blazor.server.js" LogicalName="_framework\%(Filename)%(Extension)" />
+    <EmbeddedResource Include="..\..\Browser.JS\dist\$(Configuration)\blazor.server.js" LogicalName="_framework\%(Filename)%(Extension)" />
```

Comment:
@pranavkm Is this the fix for https://github.com/aspnet/AspNetCore/issues/9402#issuecomment-483785583?

Reply:
Yup. See https://github.com/aspnet/AspNetCore/pull/9421#discussion_r275927085 - it essentially checks in the js files and adds a check to ensure that the build isn't broken. I wanted to verify things work correctly and that the code check correctly identifies if we failed to update it before I made this non-draft.

Reply:
I support this approach. It's also a workaround for needing to make NodeJS a prerequiste to building aspnetcore on all platforms we build. In some environments, we don't have NodeJS and it might be difficult to add as a new pre-req.